### PR TITLE
changes to implement variable number of directions and vector history

### PR DESCRIPTION
**Pull request by motoyama20**
Parameters are now:
1) input file path
2) output file path
3) normalization flag
4) alpha
5) vector history length
6) number of directions

Sample run command:
./gobs Main/input/straightData.csv Main/output.txt 1 1 8 4

### CODE REVIEW

File: Main/main.c [hunk 1]
```diff
@@ -64,54 +64,53 @@ int main(int argc, char *argv[])
...
-    int reduceNoiseFlag;
-    if (argv[4] == NULL)
+    if (argv[5] != NULL)
     {
-        reduceNoiseFlag = 0;
+        VEC_HIST_LEN = atoi(argv[5]);
+        printf("Setting VEC_HIST_LEN to: %d\n", VEC_HIST_LEN);
     }
-    else
+    if (argv[6] == NULL)
```
Comment (holbrook20) [inline Main/main.c]: Is this supposed to be != ?

File: Main/main.c [hunk 2]
```diff
@@ -64,54 +64,53 @@ int main(int argc, char *argv[])
     }
 
     //--------- Check normalize and reduceNoiseFlag parameters ---------
-    int normalize;
     if (argv[3] == NULL)
```
Comment (holbrook20) [inline Main/main.c]: Is this supposed to be != ?

**Review by holbrook20 — CHANGES_REQUESTED** [review]
Most of it looks good! Some questions about the command line parameters

File: Main/main.c (hunk 1 above)
Comment (motoyama20) [reply Main/main.c]: GOOD CATCH !!!!!!!

**Review by holbrook20 — APPROVED** [review]
LGTM

**Pull request merged**